src/classes/HiFiMixerSession.ts

```diff
@@ -660,7 +663,7 @@ export class HiFiMixerSession {
      * @returns If this operation is successful, returns `{ success: true, stringifiedDataForMixer: <the raw data that was transmitted to the server>}`. If unsuccessful, returns
      * `{ success: false, error: <an error message> }`.
      */
-    _transmitHiFiAudioAPIDataToServer(hifiAudioAPIData: HiFiAudioAPIData): any {
+    _transmitHiFiAudioAPIDataToServer(currentHifiAudioAPIData: HiFiAudioAPIData, previousHifiAudioAPIData?: HiFiAudioAPIData): any {
```

Comment:
i _think_ what you've done below is make the diff function unused. is that right? if so, is there any reason to keep the diff function? also, if that's right, you'll need to add some new properties here (volume threshold, etc)

Reply:
Correct,
the diff function has been removed yes
and the diff is happening directly in this function comparing the last data sent with the new one